app/coffee/ProjectManager.coffee

```diff
@@ -79,18 +83,21 @@ module.exports = ProjectManager =
 							# check the doc version first
 							RedisManager.getDocVersion doc_id, (error, version) ->
 								if error?
-									logger.error err: error, project_id: project_id, doc_id: doc_id, "error getting project doc version"
+									logger.error err: error, project_id: project_id, doc_id: doc_id, "error getting project doc version in getProjectDocs"
 									return cb(error)
 								# skip getting the doc if we already have that version
-								return cb() if version? and version is excludeVersions[doc_id]
+								if version? and version is excludeVersions[doc_id]
+									logger.error err: error, project_id: project_id, doc_id: doc_id, version: version, "skipping doc version in getProjectDocs"
```

Comment:
Is this an error? Are we even using this `excludeVersions` code pathway atm?

Reply:
We aren't using excludeVersions, but I wanted to check that this code isn't being executed unexpectedly!  Have added a comment to clarify that.